app/build.gradle

```diff
@@ -137,12 +137,12 @@ task ktlint(type: JavaExec, group: "verification") {
     description = "Check Kotlin code style."
     classpath = configurations.ktlint
     main = "com.github.shyiko.ktlint.Main"
-    args "src/**/*.kt"
+    args "--androidsrc/**/*.kt"
```

Comment:
There should be a space after android

Try running this locally

Reply:
Updated. The task keeps running, it never completes